src/cli/index.js

```diff
@@ -242,7 +242,10 @@ resetCommand
 		}
 
 		require('./reset').reset(options, function (err) {
-			if (err) { throw err; }
+			if (err) {
+				return process.exit(1);
```

Comment:
In some cases, the error and stack were thrown an additional time even though it was already printed to stdout via winston.